tools/Makefile.am

```diff
@@ -42,7 +42,58 @@ endif
 if WITH_ALPS
   OPT_ALPS = alps
 endif
-  OPT_HANDSHAKE = handshake
+  OPT_GPG = libgpg-error
```

Comment:
This shouldn't be necessary. We don't have libgcrypt included in the tools directory. Please drop.